index.js

```diff
@@ -190,10 +190,12 @@ module.exports = {
     let addonProvidedConfig = this._getAddonProvidedConfig(config);
     let shouldCompileModules = this._shouldCompileModules(config);
 
-    let providedAnnotation = config['ember-cli-babel'] && config['ember-cli-babel'].annotation;
+    let addonOptions = config['ember-cli-babel'] || {};
+    let babelOptions = config.babel || {};
 
     let options = {
-      annotation: providedAnnotation || `Babel: ${this._parentName()}`
+      annotation: addonOptions.annotation || `Babel: ${this._parentName()}`,
+      sourceMaps: addonOptions.sourceMaps || babelOptions.sourceMaps
```

Comment:
I thought `babel` *was* deprecated. At least, I remember fixing a deprecation error when I upgraded to ember-cli 2.12